lib/twilioconnection.js

```diff
@@ -267,6 +277,9 @@ class TwilioConnection extends StateMachine {
         this.emit('error', error);
         return;
       }
+
+      // Each incoming message should be treated as an incoming heartbeat
+      this._handleHeartbeat();
```

Comment:
Calling `_handleHeartbeat` for every `message.type` might result in unexpected logs from the SDK. Move this call into `heartbeat` and `msg` instead?

Reply:
Good point - I have updated now to call this only for msg and heartbeat cases.